src/client/components/AddNewRoom/AddRoom.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import './AddNewRoomStyle.css';
+
+import PropTypes from 'prop-types';
+
+import Room from './Room';
+
+const AddRoom = (props) => {
+  const { onFormHandler, inputChange, newRoom, roomId } = props; 
+  if (props.roomId) {
+    return <Room rooId={roomId} />; // this props will pass to Room component to fetch  users to add in the group
+  }
+  if (!roomId) {
+    return (
+      <form onSubmit={onFormHandler}>
+        <button type="submit">Add Room</button>
+        <input
+          type="text"
+          placeholder="Add new Room"
+          onChange={inputChange}
+          value={newRoom}
+          required
+        />
+      </form>
+    );
+  }
+};
+AddRoom.PropTypes = {
+  newRoom: PropTypes.string.isRequired,
```

Comment:
Suggestion: use `roomName` and mark it as not required since you are starting with an empty string.